source/common/http/conn_manager_impl.h

```diff
@@ -420,6 +441,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
   ConnectionManagerConfig& config_;
   ConnectionManagerStats& stats_; // We store a reference here to avoid an extra stats() call on the
                                   // config in the hot path.
+  ConnectionManagerTracingStats& tracing_stats_;
```

Comment:
I wouldn't bother with this, there is limited stat usage, I would just call through config.